### PR TITLE
:arrow_up: gofmt and goimports the project.

### PR DESCRIPTION
**Pull request by matthewhartstonge**
- Uses canonical go formatter (`gofmt`) to format code.
- Uses `goimports` to organise go imports.

This PR doesn't include the import pathing patch in #3 and will need to be merged in.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -1,32 +1,27 @@
...
 
 import (
-
 	"log"
-	"github.com/gorilla/mux"
 	"net/http"
+
+	"github.com/gorilla/mux"
+
 	"capi/api"
```
Comment (matthewhartstonge) [inline main.go]: I've split `"capi/api"` out to the bottom as I know it's a local import. 
Community import convention style seems to follow:

```
import (
	// stdlib Imports
	...

	// External Imports
	...

	// Internal Imports
	...
)

**Comment by matthewhartstonge** [comment]
Heyo, I've pushed a new commit to fix up the merge conflicts. 👌

**Comment by aciddude** [comment]
@matthewhartstonge  hey sorry I pushed before I saw this.  could you refix using the latest master branch ?

**Comment by aciddude** [comment]
@matthewhartstonge  actually, it looks like this work got done when I pushed my last commit. I followed your instructions from Telegram for FMT and IMPORTS

**Pull request closed without merging**